### PR TITLE
Adding namespace to upgrade and rollback

### PR DESCRIPTION
**Pull request by almahmoud**
Releases are only visible within one namespace at a time in Helm 3

### CODE REVIEW

**Comment by almahmoud** [comment]
Problem observed in upgrade, but added to rollback as well before it becomes a problem

File: cloudman/helmsman/clients/helm_client.py [hunk 1]
```diff
@@ -114,14 +116,17 @@ def history(self, release_name):
         data = helpers.run_list_command(["helm", "history", release_name])
         return data
 
-    def rollback(self, release_name, revision=None):
+    def rollback(self, release_name, revision=None, namespace=None):
```
Comment (nuwang) [inline cloudman/helmsman/clients/helm_client.py]: @almahmoud Namespace should be mandatory for helm3?

Comment (almahmoud) [reply cloudman/helmsman/clients/helm_client.py]: We could make it mandatory everywhere. The only place it might not make sense is list.

Comment (nuwang) [reply cloudman/helmsman/clients/helm_client.py]: Sounds good.

**Comment by nuwang** [comment]
Manually merged including namespace made mandatory.

**Pull request closed without merging**